pkgs/development/libraries/aws-sdk-cpp/default.nix

```diff
@@ -65,6 +65,14 @@ stdenv.mkDerivation rec {
   '' + lib.optionalString stdenv.hostPlatform.isMusl ''
     # TestRandomURLMultiThreaded fails
     rm aws-cpp-sdk-core-tests/http/HttpClientTest.cpp
+  '' + ''
+    # Append the $dev output to path hints in finding Aws.h to avoid
+    # having to pass `AWS_CORE_HEADER_FILE` explicitly to cmake configure
+    # when using find_package(AWSSDK CONFIG)
+    substituteInPlace cmake/AWSSDKConfig.cmake \
+      --replace 'C:/AWSSDK/''${AWSSDK_INSTALL_INCLUDEDIR}/aws/core' \
+        'C:/AWSSDK/''${AWSSDK_INSTALL_INCLUDEDIR}/aws/core"
+        "${placeholder "dev"}/include/aws/core'
```

Comment:
This is a bit confusing to follow, with the unmatching double quotes in the replacement, could you maybe add all the surrounding double quotes?

```
    substituteInPlace cmake/AWSSDKConfig.cmake \
      --replace '"C:/AWSSDK/''${AWSSDK_INSTALL_INCLUDEDIR}/aws/core"' \
        '"C:/AWSSDK/''${AWSSDK_INSTALL_INCLUDEDIR}/aws/core"
         "${placeholder "dev"}/include/aws/core"'
```

Reply:
I dropped it because I thought it added more noise. I really dislike when I see ``'"`, as now it's already a mixture nix + bash single quote + bash double quote escaping concerns.

But I will agree that with the double quotes it's more obvious that I'm replace a single string.

Reply:
Added